mappings/net/minecraft/nbt/NbtElement.mapping

```diff
@@ -1,5 +1,62 @@
 CLASS net/minecraft/class_2520 net/minecraft/nbt/NbtElement
 	COMMENT Represents an NBT element.
+	FIELD field_33250 NULL_ID B
```

Comment:
Imo these should all be changed to end with `_TYPE` rather than `_ID`, e.g. `NULL_TYPE` instead of `NULL_ID`

Reply:
Hmm, we already have a `NbtType` class I think

Reply:
Yes? It's not like we can move the fields to that class

Reply:
Should we name one `Kind` and the other `Type`? I mean in jdk they use "kind" more often than "type"

Reply:
IMO a `_TYPE` suffix would be confused with the `NbtType`s themselves, so `*_TYPE_ID` could work as a compromise? `Kind` sounds just weird to me, and it's not really used in Yarn.

Side note: `NbtType` is better than reader for that class, but that's not relevant here

Reply:
What would be going through your head to confuse these constants with an `NbtType` instance? They are ints. Also think about where these constants would be used.
```java
nbtCompound.contains("Foo", NbtElement.LIST_TYPE);
```
is way better than
```java
nbtCompound.contains("Foo", NbtElement.LIST_ID);
```
and this is just unnecessarily verbose:
```java
nbtCompound.contains("Foo", NbtElement.LIST_TYPE_ID);
```

Reply:
Lol, you commented that just as I changed them all to `TYPE_ID` :smile:

I guess `TYPE` is fine, I'll make another refactor commit. (`TYPE_ID` did match the previous usage with `NbtTypeIds` and the `nbt_type_ids` unpick key)

Reply:
Yep, I called that class `NbtTypes` before I realized we already had a class with that name, and figured it would be a bad idea to name two classes in the same area of code the same thing, for import reasons.